cfme/tests/test_rest.py

```diff
@@ -175,14 +177,12 @@ def _collection_not_in_this_version(appliance, collection_name):
 @pytest.mark.parametrize("collection_name", COLLECTIONS_ALL)
 @pytest.mark.uncollectif(
     lambda appliance, collection_name:
-        collection_name in COLLECTIONS_OMITTED or
+        collection_name == "metric_rollups" or
```

Comment:
@valaparthvi Can you get rid of `COLLECTIONS_IN_59` or change it to `COLLECTIONS_IN_510`?

Reply:
I was thinking of doing it, but then we will get a 5.11 build in the future, and there might be some collections which are a part of 5.11, but not 5.10, so I thought I'd make changes to it, once we have 5.11, in case they're required/